lib/reek/smells/utility_function.rb

```diff
@@ -53,11 +56,13 @@ def contexts # :nodoc:
       # @return [Array<SmellWarning>]
       #
       # :reek:FeatureEnvy
+      # :reek:TooManyStatements: { max_statements: 6 }
```

Comment:
Addendum: After thinking about it I came to the conclusion that the number of statements is **not** a smell here but indeed justified.

I'd leave it exactly the way it is since it is super readable.